### PR TITLE
[14.0][MIG] mail_broker, mail_broker_telegram

### PR DESCRIPTION
**Pull request by etobella**
Migration of #817

Still some work to do because a lot of work where necessary to make it work with the new JS Framework. This changes still need:

- [ ] Migration scripts
- [ ] More JS in order to send changes
- [ ] Squash of commits

Once I finish with Telegram, I will work with whatsapp

@marcelsavegnago

### CODE REVIEW

File: mail_broker/models/mail_broker_channel.py [hunk 1]
```diff
@@ -7,39 +7,32 @@
 from odoo import api, fields, models
 
 
-class MailBrokerChannel(models.Model):
-    _name = "mail.broker.channel"
-    _description = "Mail Broker Channel"
+class MailBChannel(models.Model):
```
Comment (marcelsavegnago) [inline mail_broker/models/mail_broker_channel.py]: I think there's a typo: Mail B Channel

Comment (etobella) [reply mail_broker/models/mail_broker_channel.py]: Thanks!

**Comment by marcelsavegnago** [comment]
@etobella Can you please rebase your branch?

**Comment by flachica** [comment]
If all goes well, I could contribute to this project. I need to assess the effort required to make this branch operational. It would be a great help, my dear friend @etobella, if you could clarify a few things:

* Process for incorporating my commits: Do I create a fork of your branch and have you accept pull requests? Do I create my own branch? I don't want any bureaucratic issues, authorship problems, etc. I want to do it right.
* Project status: Although I'll do my part and test it, if you could tell me if the module is already functional, that would save me some time. I'm not sure if there are still potential improvements or if they're bugs.
* As a side note, my goal is to have WhatsApp running version 14.0, and to achieve this, I'm focusing on improving all dependencies.

**Pull request closed without merging**